bower.json

```diff
@@ -17,7 +17,8 @@
     "iron-media-query": "PolymerElements/iron-media-query#^1.0.0",
     "iron-resizable-behavior": "PolymerElements/iron-resizable-behavior#2.0-preview",
     "iron-scroll-target-behavior": "PolymerElements/iron-scroll-target-behavior#^1.0.0",
-    "polymer": "Polymer/polymer#2.0-preview"
+    "polymer": "Polymer/polymer#2.0-preview",
+    "app-route": "polymerelements/app-route#^0.9.0"
```

Comment:
Why adding `app-route` here? Only the templates need `app-route` and they have their own bower.json.